Library/Formula/haxe.rb

```diff
@@ -26,7 +26,7 @@ class Haxe < Formula
   def install
     # Build requires targets to be built in specific order
     ENV.deparallelize
-    system "make", "OCAMLOPT=ocamlopt.opt"
+    system "make", "OCAMLOPT=ocamlopt.opt", ("ADD_REVISION=1" if build.head?)
```

Comment:
that gives the same RTE as you saw.
i'll make a new test tonight to launch the formula for both head and regular install and see if maybe something changed in the last month or so ...